components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -77,6 +77,9 @@ struct SiteInfo {
   bool is_content_association_possible;
   // Current tab's hostname
   string? hostname;
+  // Percentage of the content that has been truncated (0-100)
+  // Only relevant if is_content_truncated is true
+  float truncated_content_percentage;
```

Comment:
IMO we just need integer here and not float. The user already won't be able to tell exactly where in the content X% is, let alone X.YZ%

Reply:
Also is there a reason we need 2 fields for this now - both `bool is_content_truncated` and `truncated_content_percentage`? IMO this can be a single _integer_ field where `0` means not truncated and 100 means fully truncated (which we should of course never see!)

Reply:
fair enough. changed.